src/vcModals.cpp

```diff
@@ -563,6 +564,35 @@ void vcModals_DrawImageViewer(vcState *pProgramState)
   }
 }
 
+void vcModals_DrawProfile(vcState* pProgramState)
+{
+  const char* userName = pProgramState->settings.loginInfo.username;
+  const char* profile = vcString::Get("menuProfileTitle");
+
+  if (pProgramState->openModals & (1 << vcMT_Profile))
+    ImGui::OpenPopup(profile);
+
+  float width = 300.f;
+  float height = 250.f;
+  float btw = 80.f;
```

Comment:
We try to avoid shortened names like this, `buttonWidth` would be the preferred name here.